docs/root/configuration/http/http_filters/compressor_filter.rst

```diff
@@ -0,0 +1,84 @@
+.. _config_http_filters_compressor:
+
+Compressor
+==========
+Compressor is an HTTP filter which enables Envoy to compress dispatched data
+from an upstream service upon client request. Compression is useful in
+situations where large payloads need to be transmitted without
+compromising the response time.
```

Comment:
I think the tradeoff is more nuanced here; it has to do with latency, bandwidth and CPU tradeoff.

Reply:
Updated this part. Please check.

Reply:
Nice, thanks, resolved.